doc/gtdb.md

```diff
@@ -0,0 +1,137 @@
+# Working with GTDB
+
+[**GTDB**](https://gtdb.ecogenomic.org/) (Genome Taxonomy Database) ([Parks et al., 2018](https://www.nature.com/articles/nbt.4229)) is a standardized taxonomy system for bacteria and archaea. It is based on the phylogenetic trees of genomes, hence better reflecting the evolutionary relationships among microorganisms, compared to conventional taxonomy. Our research shows that GTDB shares high consistency with the [WoL](https://biocore.github.io/wol/) phylogeny (which was built using more marker genes and more expensive methods) ([Zhu et al., 2019](https://www.nature.com/articles/s41467-019-13443-4)).
```

Comment:
I think, grammatically speaking, 

> hence **better** reflecting the evolutionary relationships among microorganisms, compared to conventional taxonomy.

Could be better written:

> hence reflecting the evolutionary relationships among microorganisms **better**, compared to conventional taxonomy.

or 

> hence reflecting the evolutionary relationships among microorganisms **better than** conventional taxonomy.

Reply:
Thanks! I adopted your 1st suggestion.